ci-operator/jobs/maistra/prometheus/maistra-prometheus-maistra-2.1-presubmits.yaml

```diff
@@ -0,0 +1,52 @@
+presubmits:
+  maistra/prometheus:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^maistra-2\.1$
+    - ^maistra-2\.1-
+    cluster: build01
```

Comment:
Do you know why make update generates cluster build01 ?
It looks like a small cluster. We may need to run `make update` again.

I had larger memory requests value in maistra/istio jobs and they are on cluster: build03 from my make update.